src/core/src/bootstrap/Constants.py

```diff
@@ -312,8 +314,9 @@ class EnvLayer(EnumBackport):
     PackageClassificationOrderInStatusReporting = {
         PackageClassification.CRITICAL: 1,
         PackageClassification.SECURITY: 2,
-        PackageClassification.OTHER: 3,
-        PackageClassification.UNCLASSIFIED: 4
+        PackageClassification.SECURITY_ESM: 3,
```

Comment:
Since this order was recently changed in @feng-j678's PR, @feng-j678 was it documented? Is there a document we need to update this ordering in?

Reply:
We will need to update that document, once this change goes in

Reply:
@SathishMSFT Once this PR is merged, please work with @feng-j678 to update the doc with this change